### DISCUSSION TITLE
Imdb data access error

### DISCUSSION THREAD

**Discussion by Akhad7** [cinemagoer/cinemagoer#430] [Q&A] [answered]
Hi, I have been using this module for weeks now and it has been working fine but suddenly I am getting `IMDbDataAccessError`. I did not change the code at all and imdb website is not down so I am not sure what the problem is.

I tried running the example code and the error still occurs.
<img width="1440" alt="Screenshot 2022-12-22 at 22 20 36" src="https://user-images.githubusercontent.com/90460602/209235679-88b0fd5a-55e7-456b-90d4-04f4208451dc.png">

Any advice on what to do?

**Comment by alberanid** [comment, marked as answer]
update to the latest version from this repository.

The problem was fixed today in #426 (I will release a new version on pypi in the next days or weeks)

**Reply by Akhad7** [reply]
okay thank you!

**Comment by alberanid** [comment]
This discussion is no longer relevant due to issue https://github.com/cinemagoer/cinemagoer/issues/557: as of June 2026 Cinemagoer no longer analyzes web pages and can only rely on non-commercial datasets.
